src/DonationForms/resources/app/hooks/useFormData.ts

```diff
@@ -128,5 +150,7 @@ export default function useFormData() {
         subscriptionPeriod,
         subscriptionFrequency,
         subscriptionInstallments,
+        feeRecovery,
+        eventTicketsTotalAmount,
```

Comment:
@glaubersilva hey man! just a heads up, the vision with this useFormData hook is so gateways don't have to worry about things like `feeRecovery` and `eventTicketsTotalAmount`.  They can just grab the `amount` that will figure out the total for them.  This _should_ already be working properly through the existing `amount` variable returned here as we are interacting with the `useDonationSummaryContext()` that pulls in total amounts from other fields like event tickets and adds them to the amount.

Happy to chat more about this 😄